src/main/java/dev/openfga/sdk/api/model/StreamResultOfStreamedListObjectsResponse.java

```diff
@@ -0,0 +1,168 @@
+        this.result = result;
+    }
+
+    public StreamResultOfStreamedListObjectsResponse error(Status error) {
+        this.error = error;
+        return this;
+    }
+
+    /**
+     * Get error
+     * @return error
+     **/
+    @javax.annotation.Nullable
+    @JsonProperty(JSON_PROPERTY_ERROR)
+    @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+    public Status getError() {
+        return error;
+    }
+
+    @JsonProperty(JSON_PROPERTY_ERROR)
+    @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+    public void setError(Status error) {
+        this.error = error;
+    }
+
+    /**
+     * Return true if this Stream_result_of_StreamedListObjectsResponse object is equal to o.
+     */
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        StreamResultOfStreamedListObjectsResponse streamResultOfStreamedListObjectsResponse =
+                (StreamResultOfStreamedListObjectsResponse) o;
+        return Objects.equals(this.result, streamResultOfStreamedListObjectsResponse.result)
+                && Objects.equals(this.error, streamResultOfStreamedListObjectsResponse.error);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(result, error);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class StreamResultOfStreamedListObjectsResponse {\n");
+        sb.append("    result: ").append(toIndentedString(result)).append("\n");
+        sb.append("    error: ").append(toIndentedString(error)).append("\n");
+        sb.append("}");
+        return sb.toString();
+    }
+
+    /**
+     * Convert the given object to string with each line indented by 4 spaces
+     * (except the first line).
+     */
+    private String toIndentedString(Object o) {
+        if (o == null) {
+            return "null";
+        }
+        return o.toString().replace("\n", "\n    ");
+    }
+
+    /**
+     * Convert the instance into URL query string.
+     *
+     * @return URL query string
+     */
+    public String toUrlQueryString() {
+        return toUrlQueryString(null);
+    }
+
+    /**
+     * Convert the instance into URL query string.
+     *
+     * @param prefix prefix of the query string
+     * @return URL query string
+     */
+    public String toUrlQueryString(String prefix) {
+        String suffix = "";
+        String containerSuffix = "";
+        String containerPrefix = "";
+        if (prefix == null) {
+            // style=form, explode=true, e.g. /pet?name=cat&type=manx
+            prefix = "";
+        } else {
+            // deepObject style e.g. /pet?id[name]=cat&id[type]=manx
+            prefix = prefix + "[";
+            suffix = "]";
+            containerSuffix = "]";
+            containerPrefix = "[";
+        }
```

Comment:
This prefix and value setting is duplicated, is there any helper function we can create for this?